File: kyu8_2024/SimpleMultiplication/SimpleMultiplication.ts
// https://www.codewars.com/kata/583710ccaa6717322c000105/train/typescript

// Simple multiplication

export function simpleMultiplication(num: number): number {
  const simpleMultiplication = num % 2 === 0 ? num * 8 : num * 9;
  return simpleMultiplication;
}

File: kyu8_2024/SimpleMultiplication/SimpleMultiplication.kt
// https://www.codewars.com/kata/583710ccaa6717322c000105/train/kotlin

// Simple multiplication

fun simpleMultiplication(num: Int): Int {
    val resultMutiplication = if (num % 2 == 0) num * 8 else num * 9
    return resultMutiplication
}

fun main() {
    val result = simpleMultiplication(12)
    println(result)
}